admin/admin-filters-columns.php

```diff
@@ -66,7 +66,7 @@ protected function add_column( $columns, $before ) {
 		}
 
 		foreach ( $this->model->get_languages_list() as $language ) {
-			$columns[ 'language_' . $language->slug ] = $language->flag ? $language->flag . '<span class="screen-reader-text">' . esc_html( $language->name ) . '</span>' : esc_html( $language->slug );
+			$columns[ 'language_' . $language->slug ] = ( $language->flag ? $language->flag : sprintf( '<abbr title="%1$s">%2$s</abbr>', esc_attr( $language->name ), esc_html( $language->slug ) ) ) . '<span class="screen-reader-text">' . esc_html( $language->name ) . '</span>';
```

Comment:
I find this hard to read. I would prefer that we compute the logic and generate the output separately. Consider the following example:

```php
if ( false === $language->flag ) {
  $tag           = 'abbr';
  $attributes = array( 'title' => $language->name );
  $content    = $language->name;
}

$element = $document->createElement( $tag, $content );
foreach( $attributes as $name => $value ) {
  $element->setAttribute( $name, $value );
}

return $document->saveHTML();
```

This code use PHP's built-in `DOMDocument` as an example, but it may be switched for some less verbose HTML parsing / generating library now that we use composer to manage our dependencies.